extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -304,6 +342,19 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     });
   }
 
+  /**
+   * onclick event that fires when buy now is clicked,
+   * sends an analytics event containing product id and the conversion url.
+   * @return {boolean}
+   * @private
+   */
+  onClickBuyNow_() {
+    this.analyticsService_.triggerEvent(
+      StoryAnalyticsEvent.SHOPPING_BUY_NOW_CLICK,
+      this.element
```

Comment:
Why is `this.element` being sent?

Reply:
Good point, it was from older code when it was using data from the element in the trigger event function (before I started using auto-analytics), removed.